apps/web/src/components/AccountCard/AccountBalance.tsx

```diff
@@ -0,0 +1,25 @@
+import { AddIcon, ArrowDownIcon } from "@chakra-ui/icons";
+import { Box, Button, Flex, IconButton, Text } from "@chakra-ui/react";
+
+export const AccountBalance = () => (
+  <Box>
+    <Box marginBottom="4">
+      <Text marginBottom="1" color="gray.500" fontSize="sm">
+        Tez Balance
+      </Text>
+      <Text fontSize="2xl" fontWeight="bold">
+        2882.675746 ꜩ
+      </Text>
+      <Text color="gray.500" fontSize="md">
+        $3603.34
+      </Text>
+    </Box>
+    <Flex alignItems="center" justifyContent="space-between">
+      <IconButton borderRadius="full" aria-label="Add" icon={<AddIcon />} size="lg" />
+      <IconButton borderRadius="full" aria-label="Send" icon={<ArrowDownIcon />} size="lg" />
+      <Button borderRadius="full" colorScheme="blue" size="lg">
```

Comment:
I am not sure about the colorScheme here, I believe we should introduce our own `variant` WDYT?